src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -136,6 +137,12 @@
     @Getter
     Boolean connected_p;
 
+    private Bootstrap b;
```

Comment:
as far as I can tell, this.b is never used, and might confuse with 'b' passed as parameter to various methods.